bigbluebutton-html5/imports/ui/components/presentation/presentation-close-button/component.jsx

```diff
@@ -16,7 +16,7 @@ const ClosePresentationComponent = ({ intl, toggleSwapLayout }) => (
     aria-labelledby="closeLabel"
     aria-describedby="closeDesc"
     color="primary"
-    icon="close"
+    icon="minus"
```

Comment:
I expected that some css was to be used to make the `-` look more like `_` (move down at the very least)

Reply:
Using CSS to do that probably isn't a great idea. It might be possible to get it working initially, but it would be a big failure point in the future because you'd have to override a bunch of options. I would agree that "_" makes way more sense, but it should be done as a font icon.

Reply:
I'd be fine with a new icon. Let's keep this PR as is until the new icon is ready

Reply:
I think we are gonna need @tylercopeland 's help here.

Reply:
Upon further consideration, - looks fine :smiley_cat: